sdk/src/main/scala/com/horizen/mailbox/PrioritizedMailbox.scala

```diff
@@ -0,0 +1,17 @@
+package com.horizen.mailbox
+
+import akka.actor.ActorSystem.Settings
+import akka.dispatch.{PriorityGenerator, UnboundedPriorityMailbox}
+import com.horizen.SidechainNodeViewHolder.ReceivableMessages
+import com.typesafe.config.Config
+import scorex.core.NodeViewHolder.ReceivableMessages
+import scorex.util.ScorexLogging
+
+class PrioritizedMailbox (settings: Settings, cfg: Config) extends UnboundedPriorityMailbox (
```

Comment:
Check the usage of `UnboundedPriorityMailbox` here: https://doc.akka.io/docs/akka/current/typed/mailboxes.html
It seems we need to use `UnboundedStablePriorityMailbox` instead, because we must be sure if we get multiple messages with the same priority the order of execution is FIFO, otherwise it can lead to a random unexpected behavior.